inc/mlx90632.h

```diff
@@ -161,6 +164,18 @@
 #define MLX90632_MTYP_MEDICAL 0
 #define MLX90632_MTYP_EXTENDED 17
 
+/* Refresh rates */
+typedef enum MLX90632_MEAS {
+    MLX90632_MEAS_HZ_ERROR = -1,
+    MLX90632_MEAS_HZ_HALF = 0,
+    MLX90632_MEAS_HZ_1 = 1,
+    MLX90632_MEAS_HZ_2 = 2,
+    MLX90632_MEAS_HZ_4 = 3,
+    MLX90632_MEAS_HZ_8 = 4,
+    MLX90632_MEAS_HZ_16 = 5,
+    MLX90632_MEAS_HZ_32 = 6,
+    MLX90632_MEAS_HZ_64 = 7,
+} MLX90632_MEAS;
```

Comment:
Lowercase with underscore as separator. We can also add `_t` postfix for typdef to be a bit more POSIX alike...
```suggestion
} mlx90632_meas_t;
```